foreground_extraction.py

```diff
@@ -1,17 +1,21 @@
 import cv2
 import numpy as np
+from imutils.object_detection import non_max_suppression
```

Comment:
Move this line to next line. We want to separate “imports” and “froms”

Reply:
done